charts/longhorn/100.0.0+up1.1.2/Chart.yaml

```diff
@@ -5,7 +5,8 @@ annotations:
   catalog.cattle.io/namespace: longhorn-system
   catalog.cattle.io/os: linux
   catalog.cattle.io/provides-gvr: longhorn.io/v1beta1
-  catalog.cattle.io/rancher-version: '>= 2.6.0-0'
+  catalog.cattle.io/rancher-version: '>= 2.6.0-0 <= 2.6.100-0'
+  catalog.cattle.io/kube-version: '>=v1.16.0-r0'
```

Comment:
kube-version should not have a `v` at the beginning? Also, why do we use `-r0` as the pre-release, shouldn't it be `-0` so that something like `1.16.0-patch1` would also work?

I see that we do that with the chart kubeVersion, but I don't think there's an issue with expanding the scope of the kubeVersion field in an OOB release (the problem is usually reducing scope, since that breaks existing setups)... we should modify the `kubeVersion` annotation to also be `'>=v1.16.0-0'`

Reply:
also, kubeVersion should be capped off like `'>= v1.16.0-0, < v1.22.0-0'`